packages/medusa/src/api/routes/admin/collections/index.ts

```diff
@@ -17,6 +17,8 @@ export default (app) => {
   route.get("/:id", middlewares.wrap(require("./get-collection").default))
   route.get("/", middlewares.wrap(require("./list-collections").default))
 
+  route.post("/:id/products", middlewares.wrap(require("./update-products").default))
```

Comment:
I don't think this is very RESTy as to me this would indicate: "Add 1 new product to the collection" following regular REST semantics.

I think we should consider doing a `batch` endpoint - could be:

```
POST /collections/:id/products/batch
```

https://diegosucaria.info/handling-batch-operations-with-rest-apis/

In the future it might also be nice to have a `POST /batch/...` endpoint where the response gives you a JOB id that can be used for polling until the batch operations have been completed. Would be useful for things like product imports